netlify.toml

```diff
@@ -1,2 +1,5 @@
 [[plugins]]
-package = "@netlify/plugin-nextjs"
\ No newline at end of file
+  package = "@netlify/plugin-nextjs"
+
+[functions]
+  external_node_modules = ["sharp"]
```

Comment:
minor: file has no newline at end of file.

Reply:
Updated 👍 .